aspnet/web-pages/overview/getting-started/introducing-aspnet-web-pages-2/getting-started.md

```diff
@@ -153,31 +153,31 @@ You do not have to sign in right now to continue with this tutorial. If you do n
 
 At this point, either sign in with your Microsoft account or select **Not Now** in the lower right corner.
 
-![Sign In](getting-started/_static/image7.png)
+![Screenshot of the Windows Azure sign in dialog with the Microsoft account and organizational account sign in options.](getting-started/_static/image7.png)
 
 To begin, you'll create a blank website and add a page. In a later tutorial in this set you'll play with one of the built-in website templates.
 
 In the start window, click **New**.
 
-![WebMatrix startup screen](getting-started/_static/image8.png)
+![Screenshot of the Web Matrix Quick Start window showing the New icon highlighted with a red rectangle.](getting-started/_static/image8.png)
 
 Templates are prebuilt files and pages for different types of websites. To see all of the templates that are available by default, select the Template Gallery option.
 
-![Select Template Gallery](getting-started/_static/image9.png)
+![Screenshot of the Web Matrix Quick Start window showing the Template Gallery list item highlighted with red rectangle.](getting-started/_static/image9.png)
 
 In the **Quick Start** window, select **Empty Site** from the **ASP.NET** group and name the new site "WebPagesMovies".
 
-![WebMatrix Quick Start window with Empty Site template selected](getting-started/_static/image10.png)
+![Screenshot of the Web Matrix Quick Start window with the Empty Site template and the Site Name field highlighted with a red rectangle.](getting-started/_static/image10.png)
 
 Click **Next**.
 
 If you have signed in to your Microsoft account, you will be given the opportunity to create the site on Azure. Based on the name of your site, the default name of **WebPagesMovies.azurewebsites.net** is suggested; however, the exclamation point indicates that this name is not available on Windows Azure. For simplicity, select **Skip** to bypass creating the web site on Azure right now. Later in this series, we will publish the site to Azure.
 
-![create azure site](getting-started/_static/image11.png)
+![Screenshot of the Site from Template dialog box showing the Site Name field exclamation mark and the Skip button highlighted with a red rectangle.](getting-started/_static/image11.png)
 
 WebMatrix creates and opens the site:
 
-![New WebPagesMovies site open in WebMatrix](getting-started/_static/image12.png)
+![Screenshot of the Microsoft Web Matrix window showing the Web Pages Movies folder highlighted in light blue in the folder hierarchy.](getting-started/_static/image12.png)
```

Comment:
Don't change valid alt-text

Reply:
Alt text was modified to reflect Microsoft’s accessibility guidelines for alt text per https://review.docs.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main.